### DISCUSSION TITLE
Launched AML and for no reason I get this message

### DISCUSSION THREAD

**Discussion by SinonAsuda** [X2CommunityCore/xcom2-launcher#315] [Help] [answered]
**AML Version:** 
AML version: 1.4.1
**Description**
*Launched AML and for no reason I get this message. I played with it before, but today this message came out. Before issuing this message, I was asked to choose a game in which AML will be used. Like I opened it for the first time. Well, I chose XCOM 2 WOTC and after that there was a bug.*

**Attachments**
[AML.log](https://github.com/X2CommunityCore/xcom2-launcher/files/8380950/AML.log)
[settings.zip](https://github.com/X2CommunityCore/xcom2-launcher/files/8380964/settings.zip)

![147486438-b5b06516-006f-468b-b7ac-d6cb0c98297b](https://user-images.githubusercontent.com/102674249/160849135-4b20718a-8287-4732-998e-7364eff71a93.png)
![kKqXa4wXMWE](https://user-images.githubusercontent.com/102674249/160849145-2de8172c-49f2-4c70-b70e-d50c977fe973.jpg)

**Comment by RevZero** [comment, marked as answer]
The "Welcome"-Dialog appears when the `steam_appid.txt` file does not exist. It will be created by AML. The error message indicates that another file seems to be missing.
So the question is, why several files seem to be missing from your AML directory. I would suggest that you either download your current version [v1.4.1](https://github.com/X2CommunityCore/xcom2-launcher/releases/tag/v1.4.1) or the [latest version](https://github.com/X2CommunityCore/xcom2-launcher/releases/latest) and copy/extract all files (and overwrite existing ones) into your AML folder to repair your installation.

**Reply by SinonAsuda** [reply]
> The "Welcome"-Dialog appears when the `steam_appid.txt` file does not exist. It will be created by AML. The error message indicates that another file seems to be missing. So the question is, why several files seem to be missing from your AML directory. I would suggest that you either download your current version [v1.4.1](https://github.com/X2CommunityCore/xcom2-launcher/releases/tag/v1.4.1) or the [latest version](https://github.com/X2CommunityCore/xcom2-launcher/releases/latest) and copy/extract all files (and overwrite existing ones) into your AML folder to repair your installation.

Sorry, i don't understand what exactly i need to do. I need to download AML again? (My english is not that good)

**Reply by RevZero** [reply]
Yes, new installation in different folder or overwrite AML in existing folder with files from download. Copy settings file from old installation to new folder if you want to keep/restore previous configuration.

**Reply by SinonAsuda** [reply]
> Yes, new installation in different folder or overwrite AML in existing folder with files from download. Copy settings file from old installation to new folder if you want to keep/restore previous configuration.

Thanks. It helped.
